8/community/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.13
+FROM alpine:3.13.9
```

Comment:
This seems weird, as [3.13](https://github.com/alpinelinux/docker-alpine/blob/ed36cc54c3083ad4f1a84b2c8f63431cca2768cb/x86_64/Dockerfile) and [3.13.9](https://github.com/alpinelinux/docker-alpine/blob/ed36cc54c3083ad4f1a84b2c8f63431cca2768cb/x86_64/Dockerfile) are actually the same version

Reply:
Let me try to remove the commit, and see if it busts the cache. Perhaps it will work. Still, weird that we run into this issue.